poleno/utils/tests/test_misc.py

```diff
@@ -239,31 +238,29 @@ class FileSize(TestCase):
     def test_zero_bytes(self):
         self.assertEqual(filesize(0), u'0 bytes')
 
-    @unittest.skip(u'FIXME')
     def test_supported_sizes(self):
         self.assertEqual(filesize(1023), u'1023 bytes')
-        self.assertEqual(filesize(1024), u'1.0 kB')
-        self.assertEqual(filesize(1024*1024), u'1.0 MB')
-        self.assertEqual(filesize(1024*1024*1024), u'1.0 GB')
-        self.assertEqual(filesize(1024*1024*1024*1024), u'1.0 TB')
-        self.assertEqual(filesize(1024*1024*1024*1024*1024), u'1.0 PB')
+        self.assertEqual(filesize(1024), u'1.1 kB')
```

Comment:
@martinmacko47 V jednom PR si pre velkosti suborov pridal zaokruhlovanie `return fmt.format(round(size + 0.05, 1))`. Avsak `round` sa nesprava vzdy konzistentne pri zaokruhlovanie **float** cisel.

Reply:
@viliambalaz To bola snaha implementovat `ceil` s presnostou na jedno desatinne miesto. Ale neuspesna, kedze `1024*1024` to zaokruhli na `1.1 MB`, nie na `1.0 MB`. Vyzera, ze ked namiesto `+ 0.05` pouzijem `+ 0.04999`, tak sa to sprava rozumejsie.

Reply:
:ok: